### ISSUE TITLE
Feature Request - Unable to append to files

### ISSUE THREAD

**Issue by AndrewMore1** [Adobe-CEP/CEP-Resources#160]
Currently the CEP FS api provides us with the ability to read/write files completely, but not to make incremental reads and writes.  It's particularly bad for incremental writes, since the only solution appears to be to read in the entire existing file, append the change, and then write it back to the OS.

**Comment by zimag** [comment]
Did You try nodejs to access files?
https://nodejs.org/dist/v7.7.4/docs/api/fs.html

**Comment by codearoni** [comment]
Use the node.js FS api.

**Comment by AndrewMore1** [comment]
Thanks, that works really well.  I was initially dismissing any Node.js code, since I figured it would be deliberately prevented from working on the local FS for security reasons, which is why Window.cep has fs built in.  However it appears you're both correct and there is no restrictions on Node.js fs.

**Comment by codearoni** [comment]
The node.js runtime is spawned by the parent application, so I imagine it would inherit its permissions. An actual dev would have to confirm that though.

**Comment by AndrewMore1** [comment]
Makes sense, and the window.cep.fs stuff is probably legacy or for people who don't want to enable node.js.
